filters/include/pcl/filters/boost.h

```diff
@@ -43,6 +43,7 @@
 #ifdef __GNUC__
 #pragma GCC system_header 
 #endif
+PCL_DEPRECATED_HEADER(1, 12, "")
```

Comment:
Deprecation needs to be in v1.15 not v1.12

Reply:
Ah, thanks--I wasn't sure what version to specify. Should all of them be 1.15?

Reply:
I've changed them all to 1.15